nixos/modules/services/monitoring/prometheus/exporters.nix

```diff
@@ -197,6 +197,9 @@ in
     services.prometheus.exporters.minio.minioAccessSecret = mkDefault config.services.minio.secretKey;
   })] ++ [(mkIf config.services.rspamd.enable {
     services.prometheus.exporters.rspamd.url = mkDefault "http://localhost:11334/stat";
+  })] ++ [(mkIf config.services.nginx.enable {
+    systemd.services.prometheus-nginx-exporter.after = [ "nginx.service" ];
+    systemd.services.prometheus-nginx-exporter.bindsTo = [ "nginx.service" ];
```

Comment:
Do we really need the `BindsTo`, or is a `Requires` good enough?

I assume the prometheus exporter can (and should!) keep running, even if somebody decides to stop nginx, so monitoring notices that quicker than realizing it's not able to get data from the exporter anymore.

Reply:
Yeah you're right, I'll change it to `Requires` :+1: